src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -27,6 +27,8 @@ def serialize_event(event, parent, is_root_event=False):
         "transaction.duration": event["transaction.duration"],
         "project_id": event["project_id"],
         "parent_event_id": parent,
+        # Avoid empty string for root events
+        "parent_span_id": event["trace.parent_span"] or None,
```

Comment:
This is mostly for consistency since `parent_event_id` is `None` when there isn't a parent.